package in.co.Exception_test;

public class prac2 {
	public static void main(String[] args) {
		try {
			String name ="riya";
			System.out.println(name.charAt(2));
			
		} catch (Exception e) {
			System.out.println(e);
		}
	}
	 

}
